lib/moab/storage_object_validator.rb

```diff
@@ -22,6 +23,7 @@ class StorageObjectValidator
     NO_XML_FILES = 6
     VERSIONS_NOT_IN_ORDER = 7
     FILES_IN_VERSION_DIR = 8
+    DIRS_DETECTED = 9
```

Comment:
I *think* this may impact Stanford::StorageObjectValidator, in that it declares one more error code constant ...

Reply:
changed the order - i think it should be fine now

Reply:
might be misunderstanding, but should be ok because of the auto-offset in `Stanford::StorageObjectValidator` and because we don't ever serialize those values and read them back later (and so they just have to stay consistent for a single validation execution).

https://github.com/sul-dlss/moab-versioning/blob/4016e48235d2cd152161701e02b8319937ba28e1/lib/stanford/storage_object_validator.rb#L9

Reply:
oh right... that makes sense

Reply:
Thanks @jmartin-sul - I should have looked it up myself.  I'm thinking we should have a test for this (and there's a TODO comment to that effect) so I created #310 with that in mind.